### DISCUSSION TITLE
DevTools faild to load source map

### DISCUSSION THREAD

**Discussion by edw19** [urql-graphql/urql#1893] [Q&A] [answered]
im using nextjs with urql client, i try the use the package for devtools @urql/devtools and i have  this waring.

![image](https://user-images.githubusercontent.com/47750859/130998098-156454a0-677a-46d0-8308-6250052598ad.png)

**Comment by JoviDeCroock** [comment, marked as answer]
That's a chrome warning and relates to the bundling-strategy, not to urql afaik
